lib/drivers/driver.js

```diff
@@ -163,4 +163,42 @@ Driver.prototype.push = function (opts) {};
  */
 Driver.prototype.status = function (opts) {};
 
+/**
+ * Track local files into a git repository. Performs a post request to /track
+ * endpoint with a JSON payload in the form of:
+ *
+ *
+ *          {
+ *            "message": "Commit message",
+ *            "files": [{
+ *                "name": "README.md",
+ *                "status": "modified"
```

Comment:
Does the `status` has a meaning here ? Should it be removed ? (i would guess it is of no use here)